shared/src/commonMain/kotlin/org/hyperskill/app/profile/presentation/ProfileReducer.kt

```diff
@@ -62,7 +62,20 @@ class ProfileReducer : StateReducer<State, Message, Action> {
                             gamification = state.profile.gamification.copy(
                                 hypercoinsBalance = message.hypercoinsBalance
                             )
-                        )
+                        ),
+                        streakFreezeState = if (
+                            state.streakFreezeState is ProfileFeature.StreakFreezeState.NotEnoughGems &&
+                            state.streakFreezeState.price <= message.hypercoinsBalance
+                        )  {
+                            ProfileFeature.StreakFreezeState.CanBuy(state.streakFreezeState.streakFreezeProductId, state.streakFreezeState.price)
+                        } else if (
+                            state.streakFreezeState is ProfileFeature.StreakFreezeState.CanBuy &&
+                            state.streakFreezeState.price > message.hypercoinsBalance
+                        ) {
+                            ProfileFeature.StreakFreezeState.NotEnoughGems(state.streakFreezeState.streakFreezeProductId, state.streakFreezeState.price)
```

Comment:
Exceeds `max_line_length = 120`.

Reformat code to this:
```kotlin
streakFreezeState = if (
    state.streakFreezeState is ProfileFeature.StreakFreezeState.NotEnoughGems &&
    state.streakFreezeState.price <= message.hypercoinsBalance
) {
    ProfileFeature.StreakFreezeState.CanBuy(
        state.streakFreezeState.streakFreezeProductId,
        state.streakFreezeState.price
    )
} else if (
    state.streakFreezeState is ProfileFeature.StreakFreezeState.CanBuy &&
    state.streakFreezeState.price > message.hypercoinsBalance
) {
    ProfileFeature.StreakFreezeState.NotEnoughGems(
        state.streakFreezeState.streakFreezeProductId,
        state.streakFreezeState.price
    )
} else {
    state.streakFreezeState
}
```

Reply:
You can enable "Visual guides" with the value of "120" in your IDE

Reply:
@ivan-magda 
We can enable max_line_length setting in ktlint rules.
A the moment it is manually disabled.

Reply:
Yes, we have task about this ALTAPPS-490